pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -224,6 +224,7 @@ enum ServerError {
                          // use this error to indicate that this producer is now permanently
                          // fenced. Applications are now supposed to close it and create a
                          // new producer
+    TransactionBufferNotRecover = 26; // When transaction buffer not recover completely
```

Comment:
Shall we need to expose this error to the client-side? It's better to handle it in the broker internal.